Alert suppression options for missing suppression fields (#3205)
elastic/security-docs: docs/detections/alert-suppression.asciidoc

Alert suppression allows you to reduce the number of repeated or duplicate detection alerts created by a <<create-custom-rule,custom query>> detection rule. 

> Normally, when a rule matches multiple source events, it creates multiple alerts, one for each event. When alert suppression is configured, matching events are grouped by each unique value of a specified field, and only one alert is created for each group. You can configure alert suppression to create alerts each time the rule runs, or once within a specified time window. You can also specify multiple fields to group events by unique combinations of values.

Author: I think the edit works, and I can see how "unique values" can be seen both ways (kinda like "bimonthly," is it twice a month or every two months?). Just saying "their values" keeps it simpler, and I think the meaning still comes across.

image::images/alert-suppression-options.png[Alert suppression options,400]
--

> . Under *If a suppression field is missing*, choose how to handle events with missing suppression fields (one or more of the *Suppress alerts by* fields doesn't exist on an event):

Author: I'd like to keep the original here, because it intentionally explains what "missing fields" means in a different way than just saying "fields are missing."

> . Under *If a suppression field is missing*, choose how to handle events with missing suppression fields (one or more of the *Suppress alerts by* fields doesn't exist on an event):

Reviewer: @joepeeples I like this one you suggested here ⬆️

> * *Suppress and group alerts for events with missing fields*: Group events with missing fields together, and create one alert for each group of matching events. Missing fields are given a `null` value, which is treated as a unique value for the sake of grouping and suppressing alerts. 

Author: Slight tweak:
```suggestion
* *Suppress and group alerts for events with missing fields*: Create one alert for each group of events with missing fields. Missing fields get a `null` value, which is used to group and suppress alerts. 
```
